README.org

```diff
@@ -2,162 +2,130 @@
-   : This system uses 4 bytes per array element.
-   : -------------------------------------------------------------
-   : Array size = 80000000 (elements), Offset = 0 (elements)
-   : Memory per array = 305.2 MiB (= 0.3 GiB).
-   : Total memory required = 915.5 MiB (= 0.9 GiB).
-   : Each kernel will be executed 10 times.
-   :  The *best* time for each kernel (excluding the first iteration)
-   :  will be used to compute the reported bandwidth.
-   : -------------------------------------------------------------
-   : 
-   : OPENMP DISPLAY ENVIRONMENT BEGIN
-   :    _OPENMP='201307'
-   :   [host] OMP_CANCELLATION='FALSE'
-   :   [host] OMP_DISPLAY_ENV='TRUE'
-   :   [host] OMP_DYNAMIC='FALSE'
-   :   [host] OMP_MAX_ACTIVE_LEVELS='2147483647'
-   :   [host] OMP_NESTED='FALSE'
-   :   [host] OMP_NUM_THREADS: value is not defined
-   :   [host] OMP_PLACES='cores'
-   :   [host] OMP_PROC_BIND='spread'
-   :   [host] OMP_SCHEDULE='static'
-   :   [host] OMP_STACKSIZE='4M'
-   :   [host] OMP_THREAD_LIMIT='2147483647'
-   :   [host] OMP_WAIT_POLICY='PASSIVE'
-   : OPENMP DISPLAY ENVIRONMENT END
-   : 
-   : 
-   : Number of Threads requested = 32
-   : Number of Threads counted = 32
-   : -------------------------------------------------------------
-   : Your clock granularity/precision appears to be 1 microseconds.
-   : Each test below will take on the order of 5419 microseconds.
-   :    (= 5419 clock ticks)
-   : Increase the size of the arrays if this shows that
-   : you are not getting at least 20 clock ticks per test.
-   : -------------------------------------------------------------
-   : WARNING -- The above is only a rough guideline.
-   : For best results, please be sure you know the
-   : precision of your system timer.
-   : -------------------------------------------------------------
-   : Function    Best Rate MB/s  Avg time     Min time     Max time
-   : Copy:          104902.7     0.006141     0.006101     0.006308
-   : Scale:         106522.0     0.006039     0.006008     0.006146
-   : Add:           112215.9     0.008605     0.008555     0.008762
-   : Triad:         112097.2     0.008595     0.008564     0.008710
-   : -------------------------------------------------------------
-   : Solution Validates: avg error less than 1.000000e-06 on all three arrays
-   : Results Validation Verbose Results:
-   :     Expected a(1), b(1), c(1): 1153300692992.000000 230660145152.000000 307546849280.000000
-   :     Observed a(1), b(1), c(1): 1153300824064.000000 230660161536.000000 307546882048.000000
-   :     Rel Errors on a, b, c:     1.136495e-07 7.103091e-08 1.065464e-07
-   : -------------------------------------------------------------
+#+BEGIN_SRC sh :exports both
+ispc --version
+#+END_SRC
+#+results:
+: Intel(r) Implicit SPMD Program Compiler (Intel(r) ISPC), 1.25.3 (build  @ 20241223, LLVM 19.1.6)
 
+Here is the output from the STREAM benchmark compiled with =gcc=:
```

Comment:
This hides the actual results below a bunch of boilerpate. This should probably get moved below.